src/main/java/com/google/idea/perf/TracerView.kt

```diff
@@ -21,13 +21,18 @@ import com.intellij.openapi.actionSystem.AnActionEvent
 import com.intellij.openapi.project.DumbAwareAction
 import com.intellij.openapi.rd.attach
 import com.intellij.openapi.ui.DialogWrapper
+import com.intellij.ui.components.JBLabel
 import com.intellij.ui.components.JBPanel
 import com.intellij.ui.components.JBScrollPane
 import com.intellij.ui.components.JBTextField
+import com.intellij.util.ui.JBFont
```

Comment:
nit: A couple of these imports are unused. You can use the `Optimize Imports` action in IntelliJ to get rid of them, and there's a way to configure it to optimize imports on save.

Reply:
👍